rasa_core/policies/tf_utils.py

```diff
@@ -223,8 +223,8 @@ def _compute_time_attention(attention_mechanism, attn_inputs, attention_state,
 # noinspection PyProtectedMember
 class TimeAttentionWrapperState(
     namedtuple("TimeAttentionWrapperState",
-               tf.contrib.seq2seq.AttentionWrapperState._fields +
```

Comment:
can you please revert this file and put it into the ignored list for the style check?

This is copied and slightly modified from tensorflow and to make it easier to upgrade this in the feature it is better to not modify anything we do not need to modify here.

Reply:
alright!